custom_components/tahoma/coordinator.py

```diff
@@ -114,11 +118,18 @@ async def _async_update_data(self) -> Dict[str, Device]:
                 EventName.DEVICE_CREATED,
                 EventName.DEVICE_UPDATED,
             ]:
-                self.devices = await self._get_devices()
+                self.hass.async_create_task(
+                    self.hass.config_entries.async_reload(self._config_entry_id)
+                )
+                return None
```

Comment:
Not sure about this one. Why return  None?

Reply:
I am not 100% sure, I took it from core: https://github.com/home-assistant/core/blob/8103d9ae3c12664cdec27d742481cca28eee0796/homeassistant/components/uptimerobot/__init__.py#L79.

However my belief was that we ask the integration to reload and don't return anything in this update cycle, since it doesn't make sense. The integration will reload and refresh everything anyway.